### DISCUSSION TITLE
Minor typo in ch 11

### DISCUSSION THREAD

**Discussion by ninasiam** [rasbt/machine-learning-book#150] [General]
Hello all!

I have noticed a minor typo, most probably in the older version of the book. In pg 342, it refers to z(h) as the 1 x m dimensional feature vector. In the following lines, z(h) refers to the correct vector (matrix - vector multiplication result). 

I think it is worth mentioning, so to be corrected in the following version (if it not already :) )

**Comment by rasbt** [comment]
Thanks for your note @ninasiam , I appreciate it! If I see it correctly, this should indeed be correct in the current version if I am looking at the right spot
<img width="789" alt="Screenshot 2023-10-14 at 8 44 45 AM" src="https://github.com/rasbt/machine-learning-book/assets/5618407/f5063be9-acfc-451d-a69b-d4902bb976c6">

**Reply by ninasiam** [reply]
Yes, that's the spot! It is correct now! Thank you 😀

**Reply by rasbt** [reply]
Great! Thanks for reporting!
